src/utils/constants.ts

```diff
@@ -20,4 +20,7 @@ export const SLEEP_TIME_MS = 1000;
 
 export function sleep(ms: number): Promise<void> {
     return new Promise((resolve) => setTimeout(resolve, ms));
-}
\ No newline at end of file
+}
+
+// Decimal precision of Newton's approximations that are computed in this class.
+export const STABLE_POOLS_APPROXIMATION_PRECISION = Big(10).pow(9);
```

Comment:
Wasn't that `.pow(18)` before? Or is this a different value?

Reply:
It was 18, but I reduced it to 9 to make the computation faster - the approximations are still precise enough for the swap amounts computations.